src/ansys_sphinx_theme/latex/__init__.py

```diff
@@ -46,3 +47,36 @@ def generate_preamble(title, watermark="watermark", date=None):
     )
     template = latex_jinja_env.get_template(".")
     return template.render(variables)
+
+
+def generate_404(
+    issue_page="https://github.com/ansys/ansys-sphinx-theme/issues",
+    project_name="ansys-sphinx-theme",
+    mail_id="pyansys.support@ansys.com",
+    team_name="PyAnsys",
+):
```

Comment:
The issues page should be automatically deduced from the project's name and org that it belongs to. I think the best input arguments should be: ``org`` or ``owner``, ``project_name``, ``mail_id`` and ``team_name``.

``issue_page`` can then be created as:

```py
def generate_404(
    owner="ansys",
    project_name="ansys-sphinx-theme",
    mail_id="pyansys.support@ansys.com",
    team_name="PyAnsys",
):
    issue_page = f"https://github.com/{owner}/{project_name}/issues"
```